src/app/component/dashboard/dashboard.component.ts

```diff
@@ -1,57 +1,25 @@
 import { Component,OnDestroy,OnInit } from '@angular/core';
 import { Params, Router } from '@angular/router';
 import quizData from '../../../assets/json/data.json';
-import { QuestionService } from '../../service/question.service';
-
 @Component({
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.scss']
 })
 export class DashboardComponent implements OnInit,OnDestroy {
-  quizData = quizData;
+  quizData = {...quizData};
   title: any;
```

Comment:
please declare title type

Reply:
it was also not in use so removed it too